client/src/components/HomeScreen.tsx

```diff
@@ -97,6 +100,57 @@ const LoginScreen: React.FC = () => {
   // Support two query parameters: 'error' and 'message'
   // We use these to communicate authentication errors to the user.
   const query = new URLSearchParams(useLocation().search)
+  const {
+    register: registerEmail,
+    handleSubmit: handleSubmitEmail,
+    errors: errorsEmail,
+    setError: setErrorEmail,
+  } = useForm<{
+    email: string
+  }>()
+  const {
+    register: registerCode,
+    handleSubmit: handleSubmitCode,
+    errors: errorsCode,
+    setError: setErrorCode,
+  } = useForm<{
+    code: string
+  }>()
+  const [submittedEmail, setSubmittedEmail] = useState<string | null>(null)
+
+  const onSubmitEmail = async ({ email }: { email: string }) => {
+    const response = await fetch('/auth/jurisdictionadmin/code', {
+      method: 'POST',
+      headers: { 'Content-type': 'application/json' },
+      body: JSON.stringify({ email }),
```

Comment:
does this have CSRF protection?

Reply:
whoops, looks like i forgot to base this branch on the csrf branch, and so i forgot to add the csrf token here. if i had based it properly, i imagine this request would have failed without the token. nice catch!